LibTrixi.jl/src/api_jl.jl

```diff
@@ -49,3 +49,53 @@ function trixi_step_jl(simstate)
 
     return nothing
 end
+
+function trixi_ndims_jl(simstate)
+    mesh, _, _, _ = Trixi.mesh_equations_solver_cache(simstate.semi)
+    return ndims(mesh)
```

Comment:
For example, this will probably not work, but would have to be
```suggestion
    return Trixi.ndims(mesh)
```
since I don't see it in the `using` statement

Reply:
I thought `ndims` was just some basic julia statement.

Reply:
That's true. Maybe you're right and the method is automatically added when doing `using Trixi: Trixi`, but I would try to test this if I were you.